src/bundle/Resources/public/js/alloyeditor/src/plugins/ez-embed.js

```diff
@@ -64,7 +64,6 @@ const ZERO_WIDTH_SPACE = '&#8203;';
                 },
 
                 insertWrapper: function(wrapper) {
-                    this.editor.insertHtml(ZERO_WIDTH_SPACE);
```

Comment:
This was the fix for https://jira.ez.no/browse/EZP-29882
I think we cannot merge this.

Reply:
@dew326 sorry, but this line causes errors: in some cases, the generated XML is invalid because of zero-width space. I tried to reproduce the mentioned issue and wasn't able to do so. Seems like inline embed functionality changed: user don't provide any text. @barbaragr can you please check this one also?

Reply:
@dew326 is right, regression could occur if we merged it: https://recordit.co/gtMfFuBtyl https://jira.ez.no/browse/EZP-29882 doesn't work here.